extensions/smallrye-reactive-messaging-kafka/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/kafka/deployment/KafkaCodecProcessor.java

```diff
@@ -28,4 +35,41 @@ public void build(BuildProducer<ReflectiveClassBuildItem> reflectiveClass) {
             reflectiveClass.produce(new ReflectiveClassBuildItem(true, false, s.getName()));
         }
     }
+
+    @BuildStep
+    public void deprecateVertxProvidedSerde(BuildProducer<BytecodeTransformerBuildItem> producer) {
+        for (Class<?> vertxSerdeClass : BUILT_INS) {
```

Comment:
Minor: I would rename the BUILT_IN constant.

Reply:
Yes and I also need to register newly created Serdes as reflective classes in kafka-client.